packages/@aws-cdk/aws-elasticloadbalancing/lib/load-balancer.ts

```diff
@@ -318,6 +320,8 @@ export class LoadBalancer extends Resource implements IConnectable {
     // Keep track using array so user can get to them even if they were all supplied in the constructor
     this.listenerPorts.push(port);
 
+    // Allow connection to all instances to new listener.
```

Comment:
I'm seeing this line of code a couple of lines above this line.

```ts
     this.newInstancePort(instancePort);
```

It's named suspiciously like some of the code you just added. Could you have a look at that and see if that is maybe already doing what we need it to do?